llvm/test/CodeGen/AIE/aie2/schedule/swp/disable.ll

```diff
@@ -0,0 +1,119 @@
+; NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 4
+;
+; This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+; See https://llvm.org/LICENSE.txt for license information.
+; SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+;
+; (c) Copyright 2024 Advanced Micro Devices, Inc. or its affiliates
+
+; RUN: llc --mtriple=aie2 -O2 --stop-before=postmisched %s -o - | FileCheck %s
```

Comment:
As we are disabling pre-scheduler for some blocks, I thing it is enough to stop after pre-scheduler itself. Also, we can run just the scheduler on a MIR (instead of IR) input to see that the sequence remains unchanged.

Reply:
Agree. I was lazy and copy-edited an existing test.